lib/typekitter/cli.rb

```diff
@@ -0,0 +1,18 @@
+class Typekitter::CLI < Thor
+  package_name 'typekitter'
+  @@token = nil
+
+  desc 'save_token [value]', 'Saves the token to .typekitter-token'
+  def save_token(value)
+    @@token = Typekitter::Token.setup(value)
+    if @@token
+      puts "typekit token saved successfully"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Do not write to stdout. Use Rails' logger if you want to log.